vet.sh

```diff
@@ -70,21 +72,21 @@ elif [[ "$#" -ne 0 ]]; then
 fi
 
 # - Ensure all source files contain a copyright message.
-(! git grep -L "\(Copyright [0-9]\{4,\} gRPC authors\)\|DO NOT EDIT" -- '*.go')
+not 'git grep -L "\(Copyright [0-9]\{4,\} gRPC authors\)\|DO NOT EDIT" -- *.go'
```

Comment:
Does this still work as intended with the different quoting?

I didn't think any quote changes would be required at all.  Maybe the `eval` part needs to be tweaked if you are finding that they are (`"$@"`??)

Reply:
I switched the `eval` part to use `"$@"` and switched this line from its original form of:
```
(! git grep -L "\(Copyright [0-9]\{4,\} gRPC authors\)\|DO NOT EDIT" -- '*.go')	
```
to
```
not git grep -L "\(Copyright [0-9]\{4,\} gRPC authors\)\|DO NOT EDIT" -- '*.go'
```

But I see this in the output:
```
+ not git grep -L '\(Copyright [0-9]\{4,\} gRPC authors\)\|DO NOT EDIT' -- '*.go'
+ eval git grep -L '\(Copyright [0-9]\{4,\} gRPC authors\)\|DO NOT EDIT' -- '*.go'
++ git grep -L '(Copyright' '[0-9]{4,}' gRPC 'authors)|DO' NOT EDIT -- backoff.go balancer.go balancer_conn_wrappers.go balancer_conn_wrappers_test.go balancer_switching_test.go balancer_test.go balancer_v1_wrapper.go call.go call_test.go clientconn.go clientconn_state_transition_test.go clientconn_test.go codec.go codec_test.go dialoptions.go doc.go grpc_test.go interceptor.go picker_wrapper.go picker_wrapper_test.go pickfirst.go pickfirst_test.go preloader.go proxy.go proxy_test.go resolver_conn_wrapper.go resolver_conn_wrapper_test.go rpc_util.go rpc_util_test.go server.go server_test.go service_config.go service_config_test.go stream.go trace.go trace_test.go version.go
fatal: unable to resolve revision: [0-9]{4,}
```

Somehow the pattern to be grep'ed seems to have been broken apart when it is passed to `git grep -L`.

Reply:
When I call it as:
```
not 'git grep -L "\(Copyright [0-9]\{4,\} gRPC authors\)\|DO NOT EDIT" -- "*.go"' 
```

The output is as follows:
```
+ not 'git grep -L "\(Copyright [0-9]\{4,\} gRPC authors\)\|DO NOT EDIT" -- "*.go"'
+ eval 'git grep -L "\(Copyright [0-9]\{4,\} gRPC authors\)\|DO NOT EDIT" -- "*.go"'
++ git grep -L '\(Copyright [0-9]\{4,\} gRPC authors\)\|DO NOT EDIT' -- '*.go'
```